### ISSUE TITLE
Change y-label of Cumulative Returns Volatility plot

### ISSUE THREAD

**Issue by jmccorriston** [quantopian/pyfolio#360]
![image](https://cloud.githubusercontent.com/assets/4500507/21772698/4b8a02aa-d65a-11e6-84ae-0883d0de17f1.png)

When I originally saw these two plots, I didn't understand the difference because I was looking at the y-label. I think the y-axis label on the second plot should be 'Cumulative Returns Volatility'.

**Comment by georgh0021** [comment]
Good catch, but unfortunately its not the one-line change that it should be: both graphs are plotted using `plot_rolling_returns` in `plotting.py`. This should be an easy fix once #401 is merged though.
